### ISSUE TITLE
[DATA] CVE-2020-26140, CVE-2020-26142, CVE-2020-26143 (FragAttacks)

### ISSUE THREAD

**Issue by DerDakon** [nluedtke/linux_kernel_cves#328]
**Change Type Requested**
Remove

**CVE id number**
CVE-2020-26140
CVE-2020-26142
CVE-2020-26143

**References**
https://nvd.nist.gov/vuln/detail/CVE-2020-26140
https://nvd.nist.gov/vuln/detail/CVE-2020-26142
https://nvd.nist.gov/vuln/detail/CVE-2020-26143

**Additional context**
Given the CVE descriptions, and also the list of affected packages as provided by MITRE I think that these 3 issues do not affect Linux at all, but are exclusive to Windows and OpenBSD.

**Comment by nluedtke** [comment]
Yeah...the collect is confused because Red Hat has https://bugzilla.redhat.com/show_bug.cgi?id=1960496
